tests/Unit/Cache/Psr16Test.php

```diff
@@ -109,7 +109,8 @@ public function testSaveReturnsFalseIfDataOrMtimeCouldNotBeWritten()
 
     public function testSaveCatchesCacheExceptionAndReturnsFalse()
     {
-        $e = $this->createMock(CacheException::class);
+        // BC for PHP <=7.4, use $this->createMock(CacheException::class) in PHP 8.0+
```

Comment:
Does it even make sense mock exceptions when we do not interact with them? Also, I would expect them to be a stupid objects, so mocking should not be necessary?